.goreleaser.yaml

```diff
@@ -2,15 +2,23 @@ builds:
   - main: .
     id: "protoc-gen-validate"
     binary: "protoc-gen-validate"
+    ldflags:
```

Comment:
I am still of the opinion that build flags aren't great for versions, people are going to install these tools with `go install` and they're not going to have the version flags. 
I think changing the code is messy, but it actually solves the version problem, using build flags fixes it for only a portion of users

Reply:
youre right... back to drawing board, i really dont want a two phased release.